rclpy/src/rclpy_common/src/common.c

```diff
@@ -75,6 +103,11 @@ rclpy_convert_to_py_qos_policy(void * profile)
   set_result += PyObject_SetAttrString(pyqos_profile, "history", pyqos_history);
   set_result += PyObject_SetAttrString(pyqos_profile, "reliability", pyqos_reliability);
   set_result += PyObject_SetAttrString(pyqos_profile, "durability", pyqos_durability);
+  set_result += PyObject_SetAttrString(pyqos_profile, "lifespan", pyqos_lifespan);
+  set_result += PyObject_SetAttrString(pyqos_profile, "deadline", pyqos_deadline);
+  set_result += PyObject_SetAttrString(pyqos_liveliness, "liveliness", pyqos_liveliness);
```

Comment:
Should the first parameter to this function call and the one on line 109 below be `pyqos_profile`? I think that is the python object you want to set the attribute on and matches the pattern of everything above.